### ISSUE TITLE
Make Age 5 Great Again!

### ISSUE THREAD

**Issue by justinrusso** [DarkPacks/SevTech-Ages#2989]
This is an open discussion thread allowing people to recommend suggestions for age 5. I think we all agree at this point that age 5 needs a bit of attention. Please feel free to recommend and thumbs up/thumbs down.

**Comment by Serolar** [comment]
Curiously, a GC Rocket can become nearly useless after getting the next version. It'd be nice to be able to break down a rocket for its many valuable parts, making the following rockets cheaper, or allowing those components to be used elsewhere.

**Comment by Vercci** [comment]
(Freshly updated to 3.0.8 so let me know if I'm missing something new that would help)

I'm playing on a server and one thing that looks so painful to the server that I don't even want to bother is the creative flight modifier and creative jetpack.

If I remember right I'm going to need 6 elytra to craft them. Since that recipe for elytra via abyssalcraft was removed, and due to whatever mod removes most of the loot in world-generated chests means the end cities are empty, the only elytra I'm going to find are in the pirate ships that sometimes spawn next to end cities. That means a LOT of chunks generated just to find the 6 ships required for both crafts, and then I'll have 4 dragon heads that I won't really need otherwise.

And that's if I want the quickest route to them. If someone's crazy enough to actually want arkham-batman like flight (me) I'll have to find another pirate ship minimum, and I'll probably want my own air sigil, so that's an extra two required. There's also the other BM sigil that needs an air sigil, which is elytra locked so 9 Pirate Ships required for one person on a multiplayer server that's sort of popular. The performance hit is gonna be horrendous.

Can we get some age 5 recipes for Elytra? Make em item expensive if you want just please don't make them performance expensive.

**Comment by gijsc1** [comment]
Maybe a recipe for the mars rover schematic using some of the other schematics you can get from the same chests. The rng is terrible.

**Comment by LordHellscream** [comment]
I feel like in general ---- things were great in age 0, 1, 2 (and early age 3) because exploration is not just encouraged, but necessary. 
I sort-of (not completely) stopped playing around late age3. because things kinda "degenerated" into just another tech mod, where you amass resources then staying in your base and build the next big automation ---- I don't mind that, but it feel very.... been there done that kind of things. I felt the thing that differentiate Sevtech from the crowd is the exploration, a healthy mix of making new tech and exploration where you need to explore in order to unlock things, and to make your life easier but unlocking new challenges... etc.
I am not sure what's the best way to fix this, from a high level I feel like:
1. Integrate end-game phase of various magic and tech mods into progression. (Lots of mods were needed in early game but only touched upon skin-deep, never required to go beyond that).
2. Add late game exploration for various mods into progression of later stage. 
3. Gives player more challenges that are appropriate to player's power at the time (new mobs, drops and integrate those into progression), focus on challenges (like cool boss fight, dungeon exploration) but **not grinding**.

**Comment by sharktemplar** [comment]
> Gives player more challenges that are appropriate to player's power at the time (new mobs, drops and integrate those into progression), focus on challenges (like cool boss fight, dungeon exploration, and cool loot! -- removing all the loots in chest and not replace them is one of the worst thing about this modpack) but not grinding.
Oh God, yes please. The modpack turned into such a monotonous base-turtling micro-managing **_nightmare_** at late age 3 and onwards. Do whatever possible to give players more reasons to leave their base from this point towards age 5 planet exploration, because within this period is by far the least fun experience SevTech has to offer.

**Comment by TheBrain0110** [comment]
There were a few related suggestions that came up in #1665 relating to space-age bases still relying on stone-age processing methods for some things. So that should be included too.

**Comment by Brillik** [comment]
One of the fun side projects I ventured on (early in age 4) but could easily be implemented into age 5 - more abyssalcraft exploration - the final boss fights for that mod were amazing and could fairly easily be gated into in age 5 or sooner.  Another idea is to go further in the air-ships mod, even as far as the creation of a "steampunk" sky city.  Just some thoughts.

**Comment by Tahakk** [comment]
You don't much of them, but it would be nice to be able to smelt Galacticraft ores (like Desh Ore or Meteoric Ore and more) in more than just Stone Grill or vanilla Furnace

**Comment by Mekle001** [comment]
I've been thinking a lot about some of this, and the thing that strikes me that I've not heard, is something like a modular machine (metal former?) to upgrade the Metal Press, that is more automation friendly. It could be used for what the metal press is used for, and to fill the gaps that exist today with the tinker's forge, like a automatable bucket recipe.

**Comment by Ncrpts** [comment]
I'm still in age 4 but is there a recipe in age 5 to craft the portable bed that doesn't need the horse woodchopping to make? if not that could be handy, we are still using those beds even in age 4 when exploring or out of base.

**Comment by MuteTiefling** [comment]
So, one thing I find mildly odd is the lack of options in age 5 for making things like plates, gears, wires, etc. This hit me primarily when I started building in space and realized having things run on a conveyor belt made very little sense in null G. I'd propose adding modular machines that replicate all of the functions of the metal press. Like with others, lubricant would be required as a crafting material, but it would produce them faster.

**Comment by monodemono** [comment]
You know there's a Death Compass Singularity, right?

**Comment by Tahakk** [comment]
Yes, but you are not going to kill yourself 10k times, it is easier to make farm from mobs that spawn and these ages ;)

**Comment by MarcusKron** [comment]
Age 5: teleport and wireless energy but "ehi, i need a single Bucket recipe, lets go back to tinker smelter AGAIN"

**Comment by VirxEpicoder** [comment]
Make everything from age 0-3 the easiest and cheapest things ever to craft

**Comment by nadiayorc** [comment]
I feel like there isn't really enough guidance for some of the more complex mods in the advancements, Extra Planets has almost no wiki guides or information online, it's quite a lot of work to figure out how stuff works for someone that hasn't played the mods before. For example I didn't even know that I required thermal suits for planets after The Moon, I just found out after I was dying there slowly and had to google about it. I feel like it would be good if one of the advancements before Mars informed you that you require a thermal suit.

Prior to age 5 the advancements were generally very good in guiding you through all the mods you can make use of in that age, other than a couple that I had to eventually find out about myself. Some of which I didn't even know existed in the pack other than when I came across an item from it randomly in JEI and investigated it further. 

Other than that I'm pretty happy with it, I'm currently going through the space exploration and am sometimes having to look stuff up in random Reddit threads that I find in google and stuff due to the lack of information about Extra Planets as I've said, but that issue specifically isn't really the pack's fault.

**Comment by ImMineOfficial** [comment]
Easier way to make iron, gold and other ore blocks with an easy crafting receipe. And not every time with the immersive engineering metal press.

**Comment by daniel-dsouza** [comment]
Integrating AE2rv6 or adding extra cells for fluid storage/crafting would be nice. It would require an upgrade of Forge and CTM, as of rv6 build 6

**Comment by dman11235** [comment]
I disagree on the Compact Machines.  You do still have to make those supply lines, the only issue is that you might have to rebuild them.  You get more, well, compact.  This is a great addition in the later ages, where you have Mekanism ore processing, when you can have the complex lines that don't need to be accessed at all just set up inside another machine, where you can free form multiblock design it to your liking.

However, I would like to posit that the issue with age 5 is not this, but rather a different lack of sense of purpose.  The earlier ages had a theme going for them.  0 was the beginning, 1 was tools, 2 was magic, 3 was engineering, 4 was electronics, and 5 is...space?  No.  5 is *automation*.  I really think the way to save 5 is to actually make it *easier* to get the mystical agriculture seeds.  This would end up allowing you to automate everything with regards to the space travel, and everything else.  Making them require at most Mars (or moon), rather than Saturn, would allow you to automate the tough to find, boring to mine space minerals that you need to make the higher tier rockets.  And when you have the essence necessary to do that, it's not a problem to just grow some.  But it still requires a ton of work to get there, since you need to have the infrastructure built to deal with all of the automated mineral production.  Right now, that infrastructure and time spnt automating things with it is done *after* Saturn, or well after it would have been useful.  With this change, it would now be useful to do, and set you up nicely for the final "age", creative.  Once you finally get all of these planets explored, you can then create the final thing in this age, the creative *insert the appropriate creative unit here*, which ushers in the creative, and final age.

Obviously this owuld involve some recipe tweaks, and delay the other creative machines until another age, and require a look at how to make those again.  In my mind, you would require the end tier space minerals to make them, whatever those are (I don't know as I've never actually been to the last reaches yet :P )  This gives you an end goal on the space exploration, and probably even an end goal on the modpack.

Finally, this sort of thing would allow you to have that sense of accomplishment back.  You've done all this work on your machines, your base, your systems, and everything's working together to get you a relatively easy romp through the solar system, as a sort of 'victory lap'.  The only other thing I'd add is that there needs to be a good teleportation system in place now, because the teleposers don't work cross dimension, AS ones get too cluttered with the number of planets available, and the teleporters from Mekanism and RFTools are too expensive on non-renewable resources (cheese?  Really?  You're making it cost cheese?  And it has to be moon cheese? Expensive is fine, but let us automate it :P )

**Comment by GrumpyPirate** [comment]
I've had a blast all the way up to age 5. Each mod tree has had a wonderful amount of thought and customisation put into it by the SevTech team, and it's been a joy to play through.

That is, until space exploration comes into play, and particularly the ExtraPlanets content. The base Galacticraft process was fun, but the ExtraPlanets grind is possibly the most unimaginative process I've encountered in modded Minecraft. I suppose this is more of an issue with ExtraPlanets.

There's no way I'm going through an annoying amount of tiers' worth of what is essentially a copy/pasted process made up of differently-named heavy duty plates:
1. Spend hours inside the base crafting countless resource-heavy and over-complicated plates for rocket materials.
2. Create the next tier spacesuit to ensure you can survive some arbitrary atmosphere values.
3. Spend hours/days mining the unique resource that gates access to the next rocket/planet/space suit.
4. **Repeat x10**

It's a shame, as the above process sucked away any enjoyment or excitement that surrounded heading into space, and put me off continuing with the modpack.

Please consider removing ExtraPlanets, or if not, altering some of the rocket/plate/spacesuit recipes to be less of a dull grind.

**Comment by JonoColwell** [comment]
Suggest gating some of the Refined Storage stuff and adding in some Applied Energistics stuff

Specifically, Refined Storage is simply a much better and easier to access mod than Applied Energistics, so much so that Applied energistics may as well not be in the modpack, for example.

Refined Storage has a wireless crafting grid, Applied Energistics does not
Refined storage has EXTREMELY cheap disks that will store 2,147,483,647 of any item, Applied Energistics has 64k storage disks that will store up to 63 different item types
Refined Storage has same as above fluid storage, except that AE doesn't have limited types on fluid storage (So it's just a disk that's more expensive by a long way and stores approximately 0.03 times as much fluid)
Refined Storage is also much easier to use in general, having no channels and no item type restrictions on storage.

My suggestions would be as follows
1. Remove Quantum Storage Disks and Quantum Fluid Disks, or gate them behind age creative, not only are they not really necessary they just completely invalidate any reason to build any infrastructure for your storage disk as a single disk drive can store 20 billion items for basically zero cost. Seriously, it's 10 iron, six obsidian, a diamond chest and a bucket for effectively infinite item storage once you hit age 5.
2. Add the Wireless Crafting Terminal for Applied Energistics
3. Consider restricting Refined Storage further in some other ways, at present it completely and utterly blows AE out of the water and as AE is available at roughly the same time, is harder to use, and has LESS features, there's exactly zero reason to use it. Perhaps remove the Network Transmitter and Receiver from RS? That way the trade off would be, you can have bulk storage but can't use it interdimensionally?

**Comment by JonoColwell** [comment]
Add some use for Uranium, at present it just kind of exists, perhaps nuclearcraft or something else for age 5 power generation?

**Comment by Zarepheth** [comment]
> Add some use for Uranium, at present it just kind of exists, perhaps nuclearcraft or something else for age 5 power generation?

Uranium blocks and packed ice on either side of a thermo-electric generator make good power sources when exploring planets and other dimensions. The power generation is more than adequate for fully powering a matter transmitter, a matter receiver, and a dialer; so rockets are only needed for initial exploration.

**Comment by qq512065439** [comment]
How about OpenComputers mod ?

**Comment by daddy77gaming** [comment]
I know this is late to the party :) But i wanted to share some stuff.

I feel age 1-4 is holding your hand and perfect for getting into the pack, age 5 sort of opens the whole world to you and lets you do whatever you want however you want with some active use of JEI it's YOUR challenge to figure out what your going to be doing next. ( for example reading someone farming for hours on each planet for the next tier: that means your doing something wrong and NOT using the recourses available to you, there are a couple of auto mining options available at that age )

Immersive tech solars combined with turbine/altenator makes 24krf tick..( output power from your 6 capacitors into a relay sends 6 packets of 4096 around, the hv wire transporting it send all those around, not just 1 x 4096 ) immersive capacitors respond to redstone.. so it's very easy to toggle turbine on and off. Can easy handle decent sized ae2/refined systems AND 3 tier 2 void ore/botanical and recourse miners. Saying investing in this setup will have your power needs covered till deep into age 5. ( only deep into age 5 I set up a tier 3 kyronite solar array ( mostly to power the builder quarries on the various planets ).

Most players loosing interest in Age 5, I think have trouble playing a more kitchen sink style pack, even tho it's NOT kitchen sink Age 5 has it's goals.. Learn to use JEI, dont use the same stuff every pack you play will make you enjoy minecraft so much more. Also automate as much as you can, always. Doing some JEI searching allows you to see what kinda stuff you will need a fair bit off later on.

Game on and enjoy! Playing sevtech for the 3rd time atm.. and each playthru has been so different ( this time we added infernal mobs to the pack to make early game TF and Betweenlands interesting, and force us to progress into AS to be able to handle those infernal bosses, xD )

**Comment by voltz20** [comment]
We need a way to automate black iron with mystical agriculture, especially since the black quartz recipe is disabled. Black Iron has no easy way to be automated, besides void miners, which only has a chance to drop

**Comment by sam-kirby** [comment]
You have this in 3.1.3 @voltz20

**Comment by voltz20** [comment]
the automated engineers workbench should be added so we can automate the creation of graphite rods and vacuum tubes

**Comment by sam-kirby** [comment]
You can already create the Auto Workbench from Age 3, it can be used to create Graphite Rods and Vacuum Tubes. Please stop asking for things you already have.

**Comment by firemaster1294** [comment]
Also a little late to the party here, but I think it would be great to see Abyssalcraft extended on in age 5. My reasoning for this is that the materializer is required for the higher tier orematic devices, and this comes completely flying in out of nowhere. Additionally, these are kind of made pointless once you've developed enough of the mekanism stuff, so I would definitely like to see more done with this. I like that Abyssalcraft adds a whole other level of challenge with the plagues that you can get, which are not easily cured and can kill you even with fully upgraded armour at this point. Despite the fact Abyssalcraft is less advanced (due to its rather...old? nature), it doesn't seem to fit very well with age 5. So I would propose to either make it fit better or remove the materializer as a necessity. That said, I do like the notion of manipulating atomic structure that you can get with it. Surely there must be a way to make this fit better?

I would also recommend moving moon exploration to the modern age, just so it doesn't feel as grindy in age 5. Maybe the trigger for age 5 could be the discovery of uranium on the moon?

Lastly, as I'm sure has been mentioned many times, the space exploration needs an overhaul. Searching for titanium takes forever in the asteroid belt, and it's not exactly enjoyable. Perhaps it could be made more abundant as a surface mineral on asteroids instead of just in huge clumps? Also, perhaps the travel between planets could be staged so that you can trade materials with modern villagers that would allow you to "bypass" some planets for large amounts of expensive materials? For example, Martian villagers could sell Mercury Ores, Jupiter villagers could sell Uranus Ores, etc. For that matter, all the space villagers need to sell things other than just the lunar resources. Along with this, it would be nice to see you use the old rocket in the new rocket schematic, rather than losing most your resources with the decontructor. Also, I'd like to see uranium incorporated into rockets as a fuel additive for longer distances. Perhaps "advanced rocket fuel"?
In short:
- Titanium more abundant
- Different space villages need to sell appropriate items, not just be clones of the moon
- Villagers should provide trades for different planetary resources to speed progression
- New rockets should incorporate the old rocket in the build somehow or you should get everything back in the deconstructor
- Uranium incorporated rocket fuel

**Comment by FallingSky98** [comment]
Hey, a friend and I are currently playing through this pack and just hit age 5, was wondering if 3.1.3 is going to include a age 5 expansion or if that is still further down the track, we are loving the pack this far but there are definately some areas that need work and it would be awesome if this pack got some extra love in the late game.

**Comment by FallingSky98** [comment]
It has been mentioned previously in this thread but uranium is a massive missed oportunity rn, Big Reactors would be awesome for the late game. An expanded space travel triumph set would be nice too, similar to what we see in the Between Lands and Twilight Forest, at the moment getting the white gems for the final boss fight does feel like a bit a drag.

**Comment by Kankrumm** [comment]
Hello! Veeery late reply, but it's still open. And open things are free game (I think). 
First off, there needs to be a LOT more modular machinery. I'd say that it should get special attention in stages 3, 4, and 5. Alternatives to early game resource production is far and few in-between, and modular machinery should fix that.
Second, again, modular machinery. Input and output upgrades should be craftable so the machines can be used to their full potential. 
Third, Extra planets is, without a doubt, one of the most boring mods I have ever seen in Sevtech. Someone above said that if you manually grind for it then you're playing wrong, but that really doesn't excuse the fact that the planets are the most bland, boring, and lackluster parts of Sevtech. Spiff up the generation for them, and find a way to give players a reason to return later. Possibly refer to suggestion six?
Fourth, please nerf refined storage. I don't really care if anybody says that "both have their pros and cons" when one of them quite literally has unlimited storage and network range. A solution would be to just delete the quantum disk, or make it a creative item. I'm saying this because RS already has the pros of being extremely simple, versatile, and also no item type limit for its disks. I think they're both considerable choices enough already. No need to just blow AE out of the water with unlimited storage and access range.
Fifth, and this isn't stage 5 so you can ignore it I guess, but please change the final progression item from stage 2 to something that would directly help with progression, instead of an item that you're likely to toss the moment you obtain it.
Sixth, there needs to be better tie-ins with other mods. Many people have brought up that a bunch of magic mods were kind of tossed later, and I agree. I believe that the planets could and should have special ores from past parts in progression that are difficult/annoying to obtain. Also, maybe require some abyssal-craft materials to create or upgrade your rocket or something. It's absurdly linear and boring.
Seventh, I've seen it, so I know it can be done. Power generation options should be made available through modular machinery. Also things like steam generation, salt, and almost everything that mekanism offers should have a space-creative age alternative with modular machinery that is much more powerful. I think that modular machinery should be made one of the largest stars of the creative age, due to its versatility and ability to require large amounts of effort yet also yield amazing rewards.
Industrial Presser, Slicer (sidings), Chop-o-matic (Upgrade to mek sawmill), some kind of nuclear reactor structure (can use D-T fuel from mek, or uranium, or like any type of nuclear fuel), and maybe even a late-game blast furnace like an upgrade to the arc furnace. For the industrial presser, have 2 slots or more. One has the shape for the press, which I.E. already has, and the other will have the materials that should be pressed. A more efficient electrolytic separator, an inscriber (PLEASE), and maybe even a mystical agriculture essence converter; for example, it would turn 1 steel leaf essence into 1 steel leaf. It would be expensive, but also heavily increase the efficiency for the crops. An upgrade to the quantum compressor (can compress cobblestone better), the ability to turn ultimate ingots into almost any item in the game (like the IC2 UU-Matter), heck, even a machine that just generates compressed cobblestone, with configurable tiers (i.e. you could make quadruple compressed with RF, and then be able to switch modes and make octuple compressed for an obscenely large amount of RF).
All in all, the 5th age just feels incomplete. So many more alternatives and upgrades could've been added. I feel like modular machinery could fix this by effectively replacing every machine you've seen in the game before, becoming a multiblock, and upgrading its full capabilities. Especially with things like the ultimate singularity. The game should make you feel like a god of machinery when you finish, where your machines are capable of doing almost anything and everything. Your machines should have all the abilities of your past creations, except better and more godlike. I feel like by adding an intense overhaul to stage 5 via focusing more on modular machinery and spicing up the planetary progression would make it an extremely valuable and satisfying experience.

**Comment by halsaegusa** [comment]
I am interested in this topic. The current age 5 is boring just for monotonous space exploration.
I agree with Kankrumm's opinion. Integrating the resources obtained from space development with various existing machines to form a Modular Machinery Multi-Block seems to be quite fun.
I want to double the production efficiency through space exploration and then make the facility available to aim for the Creative Age game end.
Sorry I am not good English.

**Comment by box-admiral** [comment]
I'd just like to reiterate something I saw earlier in the thread and am currently suffering from - can there be some way to acquire Elytras without trawling through the End ad nauseum? I've got wolf boots and a sigil of haste, generated thousands of chunks, and only found one end tower with no Elytra. Age 5 does seem to be the crux of some issues.

Edit: the Elytra recipe from Abyssalcraft was removed because it was available too early - surely the recipe could be altered and reimplemented somehow?

**Comment by EntityIridescent** [comment]
This is another late reply, but I’d say coupling Advanced Rocketry with Galacticraft could make things fun. You could eliminate the need for like 8 different rockets and add structures, ore, or quests to do to incentivize space stations and interplanetary travel. I’ve made packs to play for myself where space travel and advanced rocketry is integrated with galacticraft and encouraged. Plus, you can use asteroid mining, satellites, and artifacts in the warp core to gate some planets behind other resources. For the pack I made, you started on another planet, and had to travel to the moon, Mars, the asteroid belt and to Venus to get coordinates to Earth, which was plagued by radiation, and some abandoned space shuttles spawned there with coordinates to a refuge planet. While radiation and abandoned shuttles is a bit of a different genre lol, all of that was done with planet customization and stuff from AR.
